app/controllers/friendships_controller.rb

```diff
@@ -0,0 +1,99 @@
+class FriendshipsController < ApplicationController
+  include LolConnections
+  before_action :search_summoner_for_request_action, only: [:edit]
+  before_action :require_user
+  before_action :require_same_user, only: [:edit, :decide]
+  
+ 
+	def new
+    @user = current_user
```

Comment:
You make a good point about the utility of the user variable versus @user variable and I agree. However, I was looking up the purpose of using instance variables after your first comment and the information I found summarized to: "Only use instance variables for variables that will be sent to the view." In the actions where I did not make user an instance variable it was because they were not being sent to the view.
However, since your explanation favors utility/less db access(which I am in support of) over convention, I went ahead and made the changes here:  https://github.com/danfstucky/LeagueSpec/commit/ac9e83afc4188b3a52cd57f4175c8f06c50e7e68